test/plugins/link.js

```diff
@@ -4,10 +4,14 @@ var assert = require("assert");
 
 var util = require("../util");
 var link = require("../../src/plugins/irc-events/link.js");
+const path = require("path");
 
 describe("Link plugin", function() {
 	before(function(done) {
 		this.app = util.createWebserver();
+		this.app.get("/favicon.png", function(req, res) {
+			res.sendFile(path.resolve(__dirname, "../../client/img/apple-touch-icon-120x120.png"));
```

Comment:
This is for image previews, not thumbnails. This is just a test URL that serves an existing image so the image preview passes.